lib/wallaby/driver.ex

```diff
@@ -83,6 +83,14 @@ defmodule Wallaby.Driver do
   end
 
   @doc """
+  Gets the title of the current page.
+  """
+  def title(session) do
+    resp = request(:get, "#{session.base_url}session/#{session.id}/title")
```

Comment:
Yeah it would be better to say something like `page_title`. I'll update that today